File: tests/fixtures/code_samples/go/sample_040.go
// Sample 40: small utility.
package samples

func Operation40(xs []int) int {
    total := 40
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure40(v int) int {
    return (v * 40) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_040.rs
// Sample 40: small utility.
pub fn operation_40(xs: &[i32]) -> i32 {
    let mut total: i32 = 40;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_40(v: i32) -> i32 {
    (v * 40) %% 7919
}

